openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -965,6 +994,127 @@ def validate_block_volumes_count(self, h_node, h_server, node_ip):
             "Actual:'{}'".format(
                 len(h_blockvol_list['blockvolumes']), len(gluster_block_list)))
 
+    def create_heketi_blockvolumes_in_batch(
+            self, heketi_client_node, heketi_server_url, vol_count, size=1,
+            batch_amount=8, timeout=300, interval=5, skip_cleanup=False,
+            **kwargs):
+        """Creates heketi volume in batches.
+
+        Args:
+            heketi_client_node (str): Node on which cmd has to executed.
+            heketi_server_url (str): Heketi server url.
+            vol_count (int): Count of volumes to create.
+            size (str): Volume size.
+            batch_amount (int): Amount of volumes to be created in one batch.
+            timeout (int): Timeout for entire volume count to create.
+            interval (int): Time interval between each interation.
+            skip_cleanup (bool): skip cleanup or not.
+
+        Returns:
+            list: list of heketi volume id's.
+
+        Kwargs:
+            The keys, values in kwargs are:
```

Comment:
Some parameters from this list are not supported for block volume. Use refer help from heketi-client -
```
sh-4.2# heketi-cli blockvolume create --help
Create a GlusterFS block volume

Usage:
  heketi-cli blockvolume create [flags]

Examples:
  * Create a 100GiB block volume
      $ heketi-cli blockvolume create --size=100

  * Create a 100GiB block volume specifying two specific clusters:
      $ heketi-cli blockvolume create --size=100 \
        --clusters=0995098e1284ddccb46c7752d142c832,60d46d518074b13a04ce1022c8c7193c

  * Create a 100GiB block volume requesting ha count to be 2.
    (Otherwise HA count is all the nodes on which block hosting volume reside.):
	  $ heketi-cli blockvolume create --size=100 --ha=2

  * Create a 100GiB block volume specifying two specific clusters auth enabled:
      $ heketi-cli blockvolume create --size=100 --auth \
        --clusters=0995098e1284ddccb46c7752d142c832,60d46d518074b13a04ce1022c8c7193c


Flags:
      --auth              
	Optional: Enable Authentication for block volume access
      --clusters string   
	Optional: Comma separated list of cluster ids where this volume
	must be allocated. If omitted, Heketi will allocate the volume
	on any of the configured clusters which have the available space.
	Providing a set of clusters will ensure Heketi allocates storage
	for this volume only in the clusters specified.
      --ha int            
	HA count for block volume
  -h, --help              help for create
      --name string       
	Optional: Name of volume. Only set if really necessary
      --size int          
	Size of volume in GiB

Global Flags:
  -I, --insecure-tls           
	If using TLS (HTTPS) do not verify server certificates (insecure)
      --json                   
	Print response as JSON
      --secret string          
	Secret key for specified user.  Can also be
	set using the environment variable HEKETI_CLI_KEY
  -s, --server string          
	Heketi server. Can also be set using the
	environment variable HEKETI_CLI_SERVER (the default one is http://localhost:8080)
  -C, --tls-cert stringSlice   
	If using TLS (HTTPS), specify a certificate file that can be used to verify
	the TLS connection (can be repeated)
      --user string            
	Heketi user.  Can also be set using the
	environment variable HEKETI_CLI_USER
```